### PR TITLE
feat(theme-default): add `toggleDarkMode` locale option

### PR DESCRIPTION
**Pull request by Mister-Hope**

### CODE REVIEW

**Comment by Mister-Hope** [comment]
This should fix #270

File: packages/@vuepress/theme-default/src/shared/options.ts [hunk 1]
```diff
@@ -79,6 +79,11 @@ export interface DefaultThemeLocaleData extends LocaleData {
    */
   darkMode?: boolean
 
+  /**
+   * A11y text for darkmode toggle button
+   */
+  darkmodeText?: string
```
Comment (meteorlxy) [inline packages/@vuepress/theme-default/src/shared/options.ts:85]: Should use the same camelCase as the `darkMode` option

File: packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue [hunk 2]
```diff
@@ -42,10 +42,16 @@
...
+import { computed } from 'vue'
+import { useDarkMode, useThemeLocaleData } from '../composables'
 
 const isDarkMode = useDarkMode()
 const toggleDarkMode = (): void => {
   isDarkMode.value = !isDarkMode.value
 }
+
+const title = computed(() => {
+  const themeLocale = useThemeLocaleData()
```
Comment (meteorlxy) [inline packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue:54]: Put the `use` function outside computed

Comment (meteorlxy) [reply packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue:54]: In fact, we may not need a computed value here. We should set the default value in `assignDefaultLocaleOptions`

**Review by meteorlxy — CHANGES_REQUESTED** [review]
And the docs should be updated, too.

**Comment by Mister-Hope** [comment]
Just find out I format the rest of the docs when saving, but I think that's fine, if you refuse it I will revert and recommit the docs changes.

File: packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue [hunk 3]
```diff
@@ -1,5 +1,9 @@
 <template>
-  <button class="toggle-dark-button" @click="toggleDarkMode">
+  <button
+    class="toggle-dark-button"
+    :title="title"
+    @click="themeLocale.darkModeText"
```
Comment (meteorlxy) [inline packages/@vuepress/theme-default/src/client/components/ToggleDarkModeButton.vue:5]: ... I think you should test PR locally first 😅

**Comment by Mister-Hope** [comment]
My fault, I tested before opening it, but I am lazy testing it after a refactor 🌚

**Pull request merged**